### PR TITLE
Add, fix and change some things

### PR DESCRIPTION
**Pull request by LipkeGu**
Added:
- E1 (GI)
- E2 (Conscript)
- Temperat Tileset
- Radars
- refineries
- Barracks

Fix:
- Terrain Palette Selection
- Sequences
- soviet mcv

Changed:
- Owner-style
- Preview.png

### CODE REVIEW

**Comment by Phrohdoh** [comment]
1) there is no SDK for a mod
2) this is in no way a clean environment

**Comment by LipkeGu** [comment]
updated

File: rules/structures.yaml [hunk 1]
```diff
@@ -1,13 +1,5 @@
-#
-#	ALLIED STRUCTURES
-#
-
 GACNST:
 	Inherits: ^Building
-	Buildable:
-		Queue: Building
-		BuildPaletteOrder: 1000
-		Prerequisites: ~disabled
```
Comment (abcdefg30) [inline rules/structures.yaml]: Please undo this. It was intentional. Without this you can't build the construction yards using AllTech cheat anymore.

File: rules/structures.yaml [hunk 2]
```diff
@@ -78,13 +106,41 @@ GAPOWR:
 		Amount: 200
 	ScalePowerWithHealth:
 
+NAPOWR:
```
Comment (abcdefg30) [inline rules/structures.yaml]: Same here.

File: rules/structures.yaml [hunk 3]
```diff
@@ -41,8 +31,47 @@ GACNST:
 		PauseOnLowPower: true
 	WithBuildingPlacedOverlay:
 	ProvidesCustomPrerequisite:
-		Race: allies
 		Prerequisite: structures.allies
+		Race: allies
+	Power:
+		Amount: 0
+
+NACNST:
```
Comment (abcdefg30) [inline rules/structures.yaml]: Same here.

File: rules/structures.yaml [hunk 4]
```diff
@@ -78,13 +106,41 @@ GAPOWR:
...
+	Inherits: ^Building
+	Buildable:
+		Queue: Building
+		BuildPaletteOrder: 0
+		Prerequisites: ~structures.soviets
+		Hotkey: q
+	Valued:
+		Cost: 800
+	Tooltip:
+		Name: Soviet Power Plant
```
Comment (abcdefg30) [inline rules/structures.yaml]: Can you rename it to "Tesla Reactor", please?

File: rules/structures.yaml [hunk 5]
```diff
@@ -117,23 +171,64 @@ GAPILE:
...
 	Inherits: ^Building
 	Buildable:
 		Queue: Building
 		BuildPaletteOrder: 30
+		Prerequisites: anypower, ~structures.soviets
+		Hotkey: h
+	Valued:
+		Cost: 300
+	Tooltip:
+		Name: Soviet Barracks
```
Comment (abcdefg30) [inline rules/structures.yaml]: Only "Barracks", please.

File: rules/structures.yaml [hunk 6]
```diff
@@ -154,11 +249,11 @@ GAREFN:
...
-	# FreeActor: # vxl/hva loading isn't laid out too well
-	# 	Actor: CMIN
-	# 	InitialActivity: FindResources
-	# 	SpawnOffset: -1,0
-	# 	Facing: 64
+#	FreeActor: # vxl/hva loading isn't laid out too well
+#		Actor: CMIN
+#		InitialActivity: FindResources
+#		SpawnOffset: -1,0
+#		Facing: 64
```
Comment (Phrohdoh) [inline rules/structures.yaml]: These changes are just diff noise and we gain nothing from them.
Revert.

File: rules/structures.yaml [hunk 7]
```diff
@@ -167,175 +262,130 @@ GAREFN:
...
-		Hotkey: h
+		BuildPaletteOrder: 90
+		Prerequisites: proc, ~structures.soviets
 	Valued:
-		Cost: 300
+		Cost: 1000
 	Tooltip:
-		Name: Soviet Barracks
-		Description: Produces infantry
+		Name: Soviet Radar Facility
```
Comment (abcdefg30) [inline rules/structures.yaml]: Just "Radar", please.

File: rules/structures.yaml [hunk 8]
```diff
@@ -167,175 +262,130 @@ GAREFN:
...
-		Bounds: 164,100
+		BuildPaletteOrder: 90
+		Prerequisites: proc, ~structures.allies
 	Valued:
-		Cost: 2000
+		Cost: 1000
 	Tooltip:
-		Name: Soviet Ore Refinery
-		Description: Processes ore into credits.
+		Name: Allied Communication Centre
```
Comment (abcdefg30) [inline rules/structures.yaml]: "Airforce Command Headquarters" please.

**Comment by abcdefg30** [comment]
The idle sequence of the soviet ore refinery is broken.

And it seems like sth is wrong with the airforce command headquarters as well.

**Comment by abcdefg30** [comment]
Sorry @LipkeGu, I split some parts of this PR into #3.

**Comment by abcdefg30** [comment]
Oo, the unit additions weren't bad.

**Comment by LipkeGu** [comment]
as phrohdoh says the Prereqs qas wrong... and in this case you worked allready on this i decided to delete my work on TS and RA2

**Comment by LipkeGu** [comment]
i will splitt my PR to smaller things =(

**Pull request closed without merging**